vike-react/renderer/getPageElement.tsx

```diff
@@ -5,6 +5,10 @@ import { PageContextProvider } from './PageContextProvider.js'
 import type { PageContext } from 'vike/types'
 
 function getPageElement(pageContext: PageContext): JSX.Element {
+  if (!pageContext.Page) {
+    return <></>
+  }
```

Comment:
I'm not sure this is correct to have moved this early return here? As we may have two cases to differentiate:
1. `Page` may be undefined but we may still want to render the Wrapper and the Layout, or
2. `Page` is undefined because we are running on server and SSR is disabled, in which case not rendering anything is correct. But we don't know this in this function and we shouldn't. This is why this was handled in `onRenderHtml` in the previous version instead.